src/analytics/README.rst

```diff
@@ -15,19 +15,20 @@ To install frontend-analytics into your project::
 
 Add the following configuration code to your app::
 
-    import { configureAnalytics, initializeSegment } from '@edx/analytics';
+    import { configure as configureAnalytics, SegmentAnalyticsService } from '@edx/analytics';
     import LoggingService from '@edx/frontend-logging';
 
     // Example of a configured frontend-auth api client
     import apiClient from '../config/apiClient';
     // Sample object containing app configuration
     import { configuration } from '../config/environment';
 
-    initializeSegment(configuration.SEGMENT_KEY);
-    configureAnalytics({
+    configureAnalytics(SegmentAnalyticsService, {
       loggingService: LoggingService,
-      authApiClient: apiClient,
+      httpClient: apiClient,
       analyticsApiBaseUrl: configuration.LMS_BASE_URL,
```

Comment:
These config-based things can be snagged from the brand new ConfigService!  

```
import { getConfigService } from '../config';
import { getAuthenticatedHttpClient } from '../auth';
import { getLoggingService } from '../logging';


configureAnalytics(SegmentAnalyticsService, {
  configService: getConfigService(),
  httpClient: getAuthenticatedHttpClient()
  loggingService: getLoggingService(),
});
```

Something like that.

Reply:
👍